### PR TITLE
Update code style:consistent-return

### PR DESCRIPTION
**Pull request by timursevimli**
- [ ] tests and linter show no problems (`npm t`)
- [ ] tests are added/updated for bug fixes and new features
- [x] code is properly formatted (`npm run fmt`)
- [ ] description of changes is added in CHANGELOG.md
- [ ] update .d.ts typings

### CODE REVIEW

File: lib/database.js [hunk 1]
```diff
@@ -278,7 +278,6 @@ class Database {
   async scalar(table, field, ...conditions) {
     const row = await this.row(table, [field], ...conditions);
     const values = Object.values(row);
-    if (values.length < 1) return undefined;
```
Comment (tshemsedinov) [inline lib/database.js:281]: Not related to the consistent-return, but frankly speaking we can't make decision on this before deep investigation, decompiling and profiling. I am of with both solutions.

File: lib/dbms.js [hunk 2]
```diff
@@ -80,16 +80,16 @@ const generate = async (modelPath) => {
...
-    return;
+    return void console.log('Previous version is not found in ../history');
   }
   const ps = await loadModel(previousPath);
   const folder = path.basename(previousPath);
   const date = folder.substring(0, folder.lastIndexOf('v') - 1);
   console.log(`Previous version: v${ps.database.version} (${date})`);
   if (ps.database.version >= version) {
-    console.log('You have latest model version. Migration is not needed.');
-    return;
```
Comment (tshemsedinov) [inline lib/dbms.js:92]: Please revert this

File: lib/dbms.js [hunk 3]
```diff
@@ -80,16 +80,16 @@ const generate = async (modelPath) => {
   console.log(`Generate migration: ${driver}:${name} v${version} (${now})`);
   const previousPath = await getPreviousVersion(modelPath).catch(() => {});
   if (!previousPath) {
-    console.log('Previous version is not found in ../history');
-    return;
```
Comment (tshemsedinov) [inline lib/dbms.js:84]: I don't like void in this case

Comment (tshemsedinov) [reply lib/dbms.js:84]: - console.log returns undefined in node.js
- in other platforms it may differ
- this can't turn if-void-return into single line

**Review by tshemsedinov — CHANGES_REQUESTED** [review]

File: lib/database.js (hunk 1 above)
Comment (timursevimli) [reply lib/database.js:281]: This method does not comply with the recently renewed metarhia eslint rules and an error occurs as 'async scalar method expected no return value' by linter.
In addition, the element returned from the 0th index of an array with no elements will still be undefined, so we get rid of a really useless if.

File: package.json [hunk 4]
```diff
@@ -45,7 +45,7 @@
     "@types/node": "^18.15.2",
     "@types/pg": "^8.6.6",
     "eslint": "^8.36.0",
-    "eslint-config-metarhia": "^8.1.0",
+    "eslint-config-metarhia": "^8.2.0",
     "eslint-config-prettier": "^8.7.0",
     "eslint-plugin-import": "^2.27.5",
     "eslint-plugin-prettier": "^4.2.1",
```
Comment (tshemsedinov) [inline package.json:51]:
```suggestion
    "eslint-plugin-prettier": "^5.0.0-alpha.2",
```

Comment (tshemsedinov) [reply package.json:51]: - Also update prettier to `^3.0.0`
- Not necessarily, but taking the opportunity update eslint to `^8.44.0`

Comment (tshemsedinov) [reply package.json:51]: Use `npm audit` and even `npm audit fix --force` if needed before commit.

Comment (tshemsedinov) [reply package.json:51]: Going to land after this last dependencies fixes

**Comment by timursevimli** [comment]
The dependency versions you specified were updated and the commands you specified were applied.
The "scalar" method you requested to change has also been restored.

File: package.json [hunk 5]
```diff
@@ -7,7 +7,11 @@
   "bin": {
     "metasql": "bin/cli.js"
   },
-  "files": ["lib/", "bin/", "metasql.d.ts"],
+  "files": [
+    "lib/",
+    "bin/",
+    "metasql.d.ts"
+  ],
```
Comment (tshemsedinov) [inline package.json:14]:
```suggestion
  "files": ["lib/", "bin/", "metasql.d.ts"],
```

**Comment by tshemsedinov** [comment]
Landed in https://github.com/metarhia/metasql/commit/28d937629cee3e00d2fea67939a02ddb7ce9ecfc and https://github.com/metarhia/metasql/commit/65fee19f994aa18a8adc8939b39b06b1b337506d

**Comment by tshemsedinov** [comment]
Thanks, @timursevimli ! Landed with fixes to `async scalar`

**Pull request closed without merging**